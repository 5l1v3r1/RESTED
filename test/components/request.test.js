import React from 'react';
import renderer from 'react-test-renderer';

test('Request renders correctly', () => {
  const tree = renderer.create(
    <h1>Facebook</h1>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
