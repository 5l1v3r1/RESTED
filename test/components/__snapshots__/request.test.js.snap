exports[`test Request renders correctly 1`] = `
<div>
  <h2>
    Request
  </h2>
</div>
`;
